docs/pull_request_template.md

```diff
@@ -6,8 +6,6 @@
 
 If you're unsure about a box, leave it unchecked. A maintainer will help you.
 
-- [ ] I understand README.md. I also understand that acceptance of

```

Comment:
I think we should keep `I understand README.md.` because that explains our goals, non-goals, bincompat concerns, etc.

Reply:
I think that's the case for all GitHub projects and we need not require every PR to tick a thing about it. I only added that as a special thing when we were onboarding because I thought at the time we were planning on merging nothing until tests were onboarded here.

Reply:
Compromise by leaving the statement - "Be sure you've read README.md and things" - but removing the checkbox?

Reply:
I did Casey's compromise. @StephanTLavavej is that acceptable?